packages/pwa-kit-react-sdk/package.json

```diff
@@ -40,7 +40,11 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mobify/mobify-platform-sdks.git"
+    "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+  },
+  "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/pwa-kit-react-sdk#readme",
+  "bugs": {
+    "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
   },
   "author": "dev@mobify.com",
```

Comment:
wanna also update `author`?

Reply:
Following a Slack conversation, we decided to not make changes to the `author` for now.